src/database/Databases.ts

```diff
@@ -59,7 +66,7 @@ export default class MongoDatabase {
 
   private readonly giveRepository: GiveRepository;
 
-  constructor(public uri: string) {
+  constructor(public uri: string, wantRedisCache: boolean) {
```

Comment:
`wantRedisCache` sounds odd. Perhaps name it `withRedis` or `withRedisCache` instead?